ckanext/stadtzhtheme/plugin.py

```diff
@@ -575,6 +575,13 @@ def is_supported_package_type(self, pkg_dict):
     def before_map(self, map):
         # add named route 'home' as this is removed in recent versions of CKAN
         map.connect('home', '/', controller='home', action='index')
+
+        # backporting [https://github.com/ckan/ckan/pull/4032]
+        # from CKAN 2.8: Improved search in group read and bulk page
+        # TODO: remove this controller overwrite after upgrade to CKAN 2.8
```

Comment:
The other TODO (where the logic happens) is enough. Who knows if we need another overwrite at some point, then this comment would quickly become misleading.